tests/integration/nm/bond_test.py

```diff
@@ -87,3 +106,32 @@ def _delete_bond(devname):
     nmdev = nm.device.get_device_by_name(devname)
     nm.device.deactivate(nmdev)
     nm.device.delete(nmdev)
+
+
+@mainloop_run
+def _attach_slave_to_bond(bond, slave):
+    slave_nmdev = nm.device.get_device_by_name(slave)
+    curr_slave_con_profile = nm.connection.ConnectionProfile()
+    curr_slave_con_profile.import_by_device(slave_nmdev)
+
+    slave_con_profile = nm.connection.ConnectionProfile()
```

Comment:
could you rename 'slave_con_profile' to something that helps understand what's this variable purpose (update the slave's settings - if I understood correctly) ?

Something like `slave_update_settings_connection` maybe.

Reply:
It is a variable representing an object and not a function/method, therefore naming it with a verb is not that good.
It's purpose it to represent a slave`s connection profile. I could add `new` as a prefix, but I do not know why it will be better or more readable (you will still need to read all the lines that follows).

Let me know if you find `new` any better and I'll change it.

Reply:
can't say I agree, but I don't disagree enough to block this.